Update webhooks docs for new system migration (#9311)
fishtown-analytics/docs.getdbt.com: website/docs/docs/deploy/webhooks.md

   <Constant name="dbt" /> provides a secret token that you can use to [check for the authenticity of a webhook](#validate-a-webhook). It’s strongly recommended that you perform this check on your server to protect yourself from fake (spoofed) requests.

:::info

> dbt automatically deactivates a webhook after 1,000 consecutive failed delivery attempts, or after 7 days of consecutive failures if fewer than 1,000 attempts have been made. To re-activate a deactivated webhook, use any of the following methods:

Reviewer: ```suggestion
dbt automatically deactivates a webhook after either 1,000 consecutive failed deliveries or 7 consecutive days of failed deliveries, whichever occurs first. To reactivate a webhook, use one of the following methods:
```

> - **dbt platform UI**: Update the webhook's endpoint URL. dbt automatically reactivates the webhook when the URL is changed.

Author: ```suggestion
- **dbt platform UI**: Update the webhook's endpoint URL. <Constant name="dbt" /> automatically reactivates the webhook when the URL is changed.
```

> - **dbt platform UI** &mdash; Update the webhook's endpoint URL. dbt automatically reactivates the webhook when the URL is changed.

Author: ```suggestion
- **dbt platform UI**: Update the webhook's endpoint URL. dbt automatically reactivates the webhook when the URL is changed.
```

> - **REST API**: Send a `PUT` request and set `active` to `true`, or update `client_url` to a new endpoint URL.

Author: ```suggestion
- **REST API**: Send a `PUT` request to [Update a webhook](#update-a-webhook) and set `active` to `true`, or update `client_url` to a new endpoint URL.
```

> - **REST API** &mdash; Send a `PUT` request and set `active` to `true`, or update `client_url` to a new endpoint URL.

Author: ```suggestion
- **REST API**: Send a `PUT` request and set `active` to `true`, or update `client_url` to a new endpoint URL.
```

> - **Terraform Provider** &mdash; Set `active = true` or update `client_url` in your webhook resource.

Author: ```suggestion
- **Terraform Provider**: Set `active = true` or update `client_url` in your webhook resource.
```

> - **Terraform Provider**: Set `active = true` or update `client_url` in your webhook resource.

Author: ```suggestion
- **Terraform Provider**: Set `active = true` and update `client_url` in your webhook resource.
```

> - **Terraform Provider**: Set `active = true` and update `client_url` in your webhook resource.

Reviewer: ```suggestion
- **Terraform provider**: Set `active = true` and update `client_url` in your webhook resource.
```
